src/ReturnBookControl.java

```diff
@@ -1,77 +1,75 @@
 class ReturnBookControl {
 
-	private ReturnBookUI ui;
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE state;
+	private ReturnBookUi returnBookUi;
+	private enum controlState { INITIALISED, READY, INSPECTING };

```

Comment:
Enum variables should start with a Capital letter

Reply:
adjusted